onnx/defs/schema.h

```diff
@@ -79,6 +79,11 @@ class OpSchema {
   // Sets the number of inputs, either a fixed number or a min and a max.
 
   /**
+   * The earliest operator set version which this operator was
+   * present in.
+   */
+  OpSchema& SinceVersion(int n);
```

Comment:
Do we also need "last changed in version"?

Reply:
This method definitely needs some renaming (and doc-updating); it should be "LastUpdatedVersion" or something like that. The name here comes from when I was originally planning to have the ONNX repo continuously keep all information about operators, both old ones (who have since been updated in a BC-breaking way) and the up-to-date ones.

Don and I have discussed a bit about how exactly want to go about representing history change in defs. A more traditional model is that the state of the Git repository at any point in time is the definitive description of the operator state at this point in time. So, for example, if the current ONNX operator version is 34 and you want 12, you'll have to check out the ONNX repo as it was at that version to get the canonical description of that ONNX operator state. When the only description of operators is C++ code, this is not a great state of affairs; so an important further component of this story is a normative protobuf description of the operators at some version, made available for download in some way. The good thing about this approach is that it closely matches ordinary software development practice, and it doesn't require us to change very much our model for operator development; plus, you can't "edit" history.